gallery/workbook/Global Secure Access.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
so to be clear, somewhere in the portal, you'll be opening the workbook view with with `ComponentId: "Global Secure Access"` (or `GalleryResourceType: "Global Secure Access"` and some other specific kind of resource id as the `ComponentId` value?)

Reply:
Yes, you've got it right
![image](https://github.com/microsoft/Application-Insights-Workbooks/assets/49689126/a74c4fc0-a2eb-4993-957d-6609ea351f4f)